Gemfile.lock

```diff
@@ -37,22 +38,23 @@ GEM
       rspec-core (~> 3.9.0)
       rspec-expectations (~> 3.9.0)
       rspec-mocks (~> 3.9.0)
-    rspec-core (3.9.0)
-      rspec-support (~> 3.9.0)
-    rspec-expectations (3.9.0)
+    rspec-core (3.9.2)
+      rspec-support (~> 3.9.3)
+    rspec-expectations (3.9.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
-    rspec-mocks (3.9.0)
+    rspec-mocks (3.9.1)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
-    rspec-support (3.9.0)
-    sequel (5.27.0)
+    rspec-support (3.9.3)
+    ruby2_keywords (0.0.2)
+    sequel (5.33.0)
```

Comment:
### [sequel](http://sequel.jeremyevans.net)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.27.0 → 5.33.0

[_[change-log](http://sequel.jeremyevans.net/rdoc/files/CHANGELOG.html), [source-code](https://github.com/jeremyevans/sequel)_]





<details>
<summary>Commits</summary>

A change of **75** commits. See the full changes on [the compare page](https://github.com/jeremyevans/sequel/compare/5.27.0...5.33.0).

These are the first 10 commits:
- (479a3ce) [Add example of using update with expression to cheat sheet (…](https://github.com/jeremyevans/sequel/commit/479a3ce61b24e4ef62ba91cd4331b58f1290cd5f)
- (a1a2e02) [Add any_not_empty extension, for making Dataset#any? without…](https://github.com/jeremyevans/sequel/commit/a1a2e02bcf53970173a515d8584ca1dfb033c53d)
- (5d9fcbe) [Add exclude_or_null extension, for filtering datasets where …](https://github.com/jeremyevans/sequel/commit/5d9fcbed966c8858ada75268d9f76fd9463e8f1f)
- (e3721ad) [Adjust exclude_or_null test guards and documentation after t…](https://github.com/jeremyevans/sequel/commit/e3721ad948c7d074abdac0a90c36e419948864ec)
- (5f3ef8a) [Add: DB constant definition](https://github.com/jeremyevans/sequel/commit/5f3ef8ab18912bc6fc4746a00166373496e4ef2a)
- (3206ce5) [Warn when calling Sequel::JDBC::Postgres::Dataset#with_fetch…](https://github.com/jeremyevans/sequel/commit/3206ce599ad5156962175addfc8cdf1faafaf896)
- (77eb5f9) [Force pg <1.2 when using sequel_pg when testing on Travis](https://github.com/jeremyevans/sequel/commit/77eb5f929c5cf753d1b1d1522f2f0f7796de9e46)
- (d27e728) [Fix spec failure on Ruby 2.7](https://github.com/jeremyevans/sequel/commit/d27e728468456acfcce076cdc88b95bf95e1427c)
- (bca31dd) [Parenthesize endless ranges to avoid warnings on Ruby 2.7](https://github.com/jeremyevans/sequel/commit/bca31dd53b66130a83f9202a7c89df392d933690)
- (00c65c0) [Add Ruby 2.7 to Travis](https://github.com/jeremyevans/sequel/commit/00c65c0ed5ee62f5849601f6dc3bd00efa83cf49)


</details>